coverage/ctracer/tracer.c

```diff
@@ -981,9 +981,10 @@ CTracer_call(CTracer *self, PyObject *args, PyObject *kwds)
 static PyObject *
 CTracer_start(CTracer *self, PyObject *args_unused)
 {
+    assert(atomic_load_int(&self->started) == FALSE);
```

Comment:
This assert would fail without https://github.com/nedbat/coveragepy/commit/9533be81e553725110fda9c6ee0887de45225ccb while running under pytest-run-parallel